lib/thrift.ex

```diff
@@ -146,11 +146,9 @@ defmodule Thrift do
     [:bool, :i8, :i16, :i32, :i64, :binary, :double, :byte, :string]
   end
 
+  # A struct for handling IEEE-754 NaN values.
   defmodule NaN do
-    @moduledoc """
-    A struct for handling IEEE-754 NaN values.
-    """
-
+    @moduledoc false
```

Comment:
This is the only module in this PR I'm a bit iffy on. If someone gets `%Thrift.NaN{}` showing up in their stacktraces, they'll be inclined to go check our docs and won't find this. Could we leave this one in?